src/setup-java.ts

```diff
@@ -1,10 +1,20 @@
 import * as core from '@actions/core';
 import * as installer from './installer';
 import * as auth from './auth';
+import * as gpg from './gpg';
 import * as path from 'path';
 
+const DEFAULT_ID = 'github';
+const DEFAULT_USERNAME = 'GITHUB_ACTOR';
+const DEFAULT_PASSWORD = 'GITHUB_TOKEN';
+const DEFAULT_GPG_PRIVATE_KEY = undefined;
+const DEFAULT_GPG_PASSPHRASE = 'GPG_PASSPHRASE';
+
 async function run() {
   try {
+    // Set secrets before use
+    core.setSecret('gpg-private-key');
```

Comment:
I think you need to do this below and use the `gpgPrivateKey` variable. `setSecret` masks out values you want hidden so this would currently mask the text `gpg-private-key`.

Reply:
Good callout. I could have sworn that it wanted the name of the variable or something. I will take another look at the docs and implement this fix as soon as I can!